src/main/scala/cromwell/binding/types/WdlFileType.scala

```diff
@@ -1,9 +1,5 @@
 package cromwell.binding.types
 
-import java.io.File
-
 case object WdlFileType extends WdlType {
-  def isCompatible(value: Any) = value.isInstanceOf[File]
-
   override def toString: String = "File"
```

Comment:
Discussed in person, this method will be renamed as `toString` is intended more for generating a displayable representation than delivering a specific semantic meaning.